ganttproject/src/net/sourceforge/ganttproject/GPTreeTableBase.java

```diff
@@ -905,12 +911,20 @@ TableHeaderUiFacadeImpl getTableHeaderUiFacade() {
   protected TableColumnExt newTableColumnExt(int modelIndex) {
     TableColumnExt result = new TableColumnExt(modelIndex);
     Class<?> columnClass = getTreeTableModel().getColumnClass(modelIndex);
-    TableCellRenderer renderer = createCellRenderer(columnClass);
+    String columnName = getTreeTableModel().getColumnName(modelIndex).toLowerCase();
```

Comment:
This class is abstract and is shared between Gantt chart and Resource chart. Please move all checks which are related to tasks into GanttTreeTable